### ISSUE TITLE
PHP8 Support

### ISSUE THREAD

**Issue by toddb02** [cronofy/cronofy-php#110]
Just curious as to when/if there will by PHP8 support.

```
Problem 1
    - cronofy/cronofy[v1.4.0, ..., v1.x-dev] require php ^7.1 -> your php version (8.0.7) does not satisfy that requirement.
   
```

Thanks in advance.

**Comment by AdamWhittingham** [comment]
Hi @toddb02,

That's a very well-timed question! We have merged changes for our CI and composer.json to test & support PHP 8 yesterday. We'll be releasing a new version of the SDK in the next day or two, once HMAC PR (#108) is merged, so support is imminent.

**Comment by toddb02** [comment]
Thanks, @AdamWhittingham!

**Comment by AdamWhittingham** [comment]
This is now available in [v1.5.0](https://packagist.org/packages/cronofy/cronofy#v1.5.0)!
If you have any problems then please let us know via support@cronofy.com.
